spec/support/medium_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.shared_examples "a medium" do
+
+  describe "model validations" do
+    it "must have a name" do
+      medium = described_class.new
+      expect(medium).to_not be_valid
+      expect(medium.errors.keys).to include :name
+    end
+  end
```

Comment:
Are there any other validations we could put on this model class? I'm not sure if the author/director/writer field is required, but it does seem sensible that it would be.

Perhaps we could also validate that a medium with a negative ranking is not valid?